### PR TITLE
CR-1-create-navigation

### LINKED ISSUE #1: CR-1 Create navigation

### PR DESCRIPTION
**Pull request by ksawerykasp**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          <span class="navbar__title">coffeeroasters</span>
+        </div>
+        <nav class="navbar__toggleMenu">
+          <ul class="navbar__list">
+            <li class="navbar__list-item">home</li>
+            <li class="navbar__list-item">about us</li>
+            <li class="navbar__list-item">create your plan</li>
+          </ul>
+        </nav>
+        <div class="navbar__toggleButton">
```
Comment (Skolaczk) [inline index.html]: It should be `button` and rename it to `navbar__hamburger-button`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="page-container">
+      <header class="navbar">
+        <div class="navbar__logo-container">
+          <img
+            class="navbar__img"
+            src="images/coffeeIcon.svg"
+            alt="coffeeIcon"
```
Comment (Skolaczk) [inline index.html]: In alt attribute should not use word like image, icon etc. Use just `coffee` or `coffee beans`. And do not use camelCase here

File: index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        </div>
+        <nav class="navbar__toggleMenu">
+          <ul class="navbar__list">
+            <li class="navbar__list-item">home</li>
+            <li class="navbar__list-item">about us</li>
+            <li class="navbar__list-item">create your plan</li>
+          </ul>
+        </nav>
+        <div class="navbar__toggleButton">
+          <div class="navbar__toggleButton-icon"></div>
```
Comment (Skolaczk) [inline index.html]: It should be a `span` and rename it to `navbar__button-line`

File: style.css [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+    display: none;
+}
+
+.navbar__toggleButton
+{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+    width: 1.3rem;
```
Comment (Skolaczk) [inline style.css]: For styling width and height you can use `px`

File: style.css [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+
+.navbar__list
+{
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    font-size: 1.5rem;
+    padding-left: 0;
+    padding-top: 6rem;
+    font-family: 'Fraunces', serif;
```
Comment (Skolaczk) [inline style.css]: Add fonts to variables and set the `body` to the most frequent font

File: style.css [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+        display: none;
+    }
+    .navbar__toggleMenu
+    {
+        all: unset;
+    }
+    .navbar__list
+    {
+        all: unset;
+        list-style: none;
```
Comment (Skolaczk) [inline style.css:166]: Remove `list-style: none`, you already set it in 95 line

Comment (ksawerykasp) [reply style.css:166]: "all: unset;"

**Review by Skolaczk — APPROVED** [review]

**Pull request merged**